docs/pages/jpsreport/jpsreport_profiles.md

```diff
@@ -0,0 +1,38 @@
+---
+title:  Density profiles
+keywords: analysis
+tags: [jpsreport, jpsreport_tutorial]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: This page shows how to get profiles of density, velocity and specific flow.
+permalink: jpsreport_profiles.html
```

Comment:
I would remove this page for this PR.

Reply:
I unpublished the page. Can be republished again when the issue is solved.